net-libs/libyang/libyang-1.0.184.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
```

Comment:
cmake-utils is deprecated (https://devmanual.gentoo.org/eclass-reference/cmake-utils.eclass/index.html)

Reply:
I tried it on my stable Gentoo and it fails to build (something related with `BDEPENDS`), then after fixing that it complains about some missing functions. I did not want to spend more time with this so I commited as-is.

Feel free to try it and commit here if you figured it out, otherwise I'll leave this for later.